core-blocks/latest-comments/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import { DEFAULT_CONTROLS } from 'editor/components/block-alignment-toolbar';
+import './style.scss';
+import edit from './edit';
+
+export const name = 'core/latest-comments';
+
+export const settings = {
+	title: __( 'Latest Comments' ),
+
+	description: __( 'Show a list of your site’s most recent comments.' ),
+
+	icon: 'list-view',
+
+	category: 'widgets',
+
+	keywords: [ __( 'recent comments' ) ],
+
+	supports: {
+		html: false,
+	},
+
+	getEditWrapperProps( attributes ) {
+		const { align } = attributes;
+
+		if ( DEFAULT_CONTROLS.includes( align ) ) {
```

Comment:
Related is
https://github.com/WordPress/gutenberg/issues/7908#issuecomment-405667770; before this the code copied what many other elements did and ignored centre-alignment, which caused the centre alignment option to do nothing.

I feel like it's better to do this as a starting point here and convert everything else over in #7908. This would be the way we'd want things to go, right?

Reply:
> This would be the way we'd want things to go, right?

Maybe, maybe not. :) But it deserves its own PR. I'd undo here.